.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  "extends": "airbnb",
+  "rules": {
+    "react/react-in-jsx-scope": "off",
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],}
+};
```

Comment:
Empty line is missing here.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file